docs/sources/mq.md

```diff
@@ -2,6 +2,10 @@
 
 This event source is meant to be used as a Container Source with a Knative cluster to consume messages from IBM Message Queue and send them to a Knative service/function.
 
+## Prerequisite(s)
+
+- IBM MQ
+
```

Comment:
Not part of our public facing documentation, can be removed.

Reply:
@odacremolbap are you meaning to delete the whole file?